lib/plugins/wallet/zcashd/zcashd.js

```diff
@@ -93,7 +93,10 @@ function sendCoins (account, tx, settings, operatorId) {
 
 function newAddress (account, info, tx, settings, operatorId) {
   return checkCryptoCode(info.cryptoCode)
-    .then(() => fetch('getnewaddress'))
+    .then(() => fetch('z_listaccounts'))
+    .then(accountsRes => _.size(accountsRes) === 0 ? fetch('z_getnewaccount') : Promise.resolve(_.first(accountsRes)))
```

Comment:
Any particular reason to use `_.size(...) === 0` instead of `_.empty(...)` ? Very minor nitpick, but I think the latter is easier to read and better conveys the intent.